### PR TITLE
Staging to prod deploy

### PR DESCRIPTION
**Pull request by amekelburg**

### CODE REVIEW

File: app/views/layouts/_weglot.html.erb [hunk 1]
```diff
@@ -0,0 +1,43 @@
+<script type="text/javascript" src="https://cdn.weglot.com/weglot.min.js"></script>
+<script>
+    Weglot.initialize({
+        api_key: 'wg_998f62092305ff602e8efd215c8fd0633',
```
Comment (amekelburg) [inline app/views/layouts/_weglot.html.erb:4]: does this need to be more private? it's a public repo.

Comment (amekelburg) [reply app/views/layouts/_weglot.html.erb:4]: (env var)

File: app/views/layouts/lookup.html.erb [hunk 2]
```diff
@@ -47,6 +47,46 @@
     });
   </script>
   <%= yield :head %>
+<script type="text/javascript" src="https://cdn.weglot.com/weglot.min.js"></script>
+<script>
+    Weglot.initialize({
+        api_key: 'wg_998f62092305ff602e8efd215c8fd0633'
```
Comment (amekelburg) [inline app/views/layouts/lookup.html.erb:53]: same here - also can this just render the partila layout/_weglot so the code isn't repeated?

File: app/views/registrants/show.html.haml [hunk 3]
```diff
@@ -1,10 +1,12 @@
+= render :partial => 'layouts/weglot'
+
 - if !@registrant || @registrant.short_form?
   .intro-message
     %p=t('txt.registration.intro_message')
   
 %h1= t('txt.registration.step_1_header')
 
-#other_language_link
+#other_language_link{ style: 'display: none;' }
```
Comment (amekelburg) [inline app/views/registrants/show.html.haml:9]: is the mechanism for changing locales changing?

**Review by amekelburg — COMMENTED** [review]
@rock-the-vote probably the API key should be in an env var?

File: app/views/layouts/_weglot.html.erb (hunk 1 above)
Comment (rock-the-vote) [reply app/views/layouts/_weglot.html.erb:4]: Yeah, I'd move IF we decide to use this tool.

File: app/views/registrants/show.html.haml (hunk 3 above)
Comment (rock-the-vote) [reply app/views/registrants/show.html.haml:9]: We can't have both cause they conflict so hiding the existing locale changer within the ui.

**Pull request merged**